snippets/csharp/System.Linq/Enumerable/Comparers/Comparers.csproj

```diff
@@ -1,9 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net6.0;net472</TargetFrameworks>
-    <StartupObject>Program</StartupObject>
+    <TargetFrameworks>net8.0;net472</TargetFrameworks>
```

Comment:
Pinging @gewarren to be sure, but I think our framework samples are using net48 now.